htdocs/PI/write/PIWriteRequest.php

```diff
@@ -344,39 +346,12 @@ private function validateWithService($objectType,$objectProperty,$propertyValue,
      * @throws \Exception
      */
     private function updateEntity(Site $siteService) {
+      //We don't currently allow any access to unauthenticated users, so for now
+      //we can simply refuse unauthenticated users. In the future we could look
+      //to authenticate everything except 'GET' requests and then process 'GETs'
+      //on a case by case basis.
```

Comment:
Is there an issue for this?

Reply:
No - because we don't currently support GET requests. This only becomes an issue if we seek to support GET requests - as all other types will require Authorisation (and so authentication)